lib/node_modules/@stdlib/fs/append-file/README.md

```diff
@@ -0,0 +1,255 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# Append File
+
+> Append data to a file.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var appendFile = require( '@stdlib/fs/append-file' );
+```
+
+#### appendFile( file, data\[, options], clbk )
+
+Asynchronously appends `data` to a `file`.
+
+```javascript
+var join = require( 'path' ).join;
+
+var fpath = join( __dirname, 'examples', 'fixtures', 'file.txt' );
+
+appendFile( fpath, 'beep boop\n', onAppend );
+
+function onAppend( error ) {
+    if ( error ) {
+        console.log( error instanceof Error );
+        // => false
+    }
+}
+```
+
+The `data` argument may be either a `string` or a [`Buffer`][@stdlib/buffer/ctor].
+
+```javascript
+var join = require( 'path' ).join;
+var string2buffer = require( '@stdlib/buffer/from-string' );
+
+var fpath = join( __dirname, 'examples', 'fixtures', 'file.txt' );
+
+appendFile( fpath, string2buffer( 'beep boop\n' ), onAppend );
+
+function onAppend( error ) {
+    if ( error ) {
+        console.log( error instanceof Error );
+        // => false
+    }
+}
+```
+
+The function accepts the same `options` and has the same defaults as [`fs.appendFile()`][node-fs].
+
+#### appendFile.sync( file, data\[, options] )
+
+Synchronously appends `data` to a `file`.
+
+```javascript
+var join = require( 'path' ).join;
+
+var fpath = join( __dirname, 'examples', 'fixtures', 'file.txt' );
+
+try {
+    appendFile.sync( fpath, 'beep boop\n' );
+} catch ( error ) {
+    console.log( error.message );
+}
+```
+
+The function accepts the same `options` and has the same defaults as [`fs.appendFileSync()`][node-fs].
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+## Notes
+
+<!-- run-disable -->
```

Comment:
I suggest basing this note much closer to https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/fs/read-file#notes. One thing to note is that we do not disable ESLint rules within the examples themselves. To disable lint rules, you need to do `<!-- eslint-disable node/no-sync -->` before the codeblock.

Reply:
Noted and fixed